ui/guest/src/controls/rte.ts

```diff
@@ -152,7 +155,10 @@ export function initTinyMCE(
           e.preventDefault();
         }
       });
-    }
-  });
+    },
+    ...rteConfig
```

Comment:
We must make sure that certain props prevail — like setup and possibly other important behavioural props — hence they must be after the `...rteConfig` statement